libs/pages/application/src/lib/ui/page-settings-general/page-settings-general.tsx

```diff
@@ -116,62 +121,75 @@ export function PageSettingsGeneral({
                   jobType={job.job_type === 'CRON' ? ServiceTypeEnum.CRON_JOB : ServiceTypeEnum.LIFECYCLE_JOB}
                   organization={organization}
                 />
-                {isJobGitSource(job.source) ? (
-                  <>
+                <Section className="gap-4 mt-10">
```

Comment:
use `space-y-10` on `<form>` instead of `mt-10` everywhere

Reply:
https://github.com/Qovery/console/pull/1271/commits/6ed2bb30c904bff048362c528c5230e7bec1aff4